Pyrado/pyrado/algorithms/step_based/value_based.py

```diff
@@ -44,7 +44,7 @@
 from pyrado.utils.input_output import print_cbt_once
 
 
-class ValueBased(Algorithm, ABC):
+class ValueBased(Algorith, ABC):
```

Comment:
This looks wrong

Reply:
Ah, that happened after I removed the Exposed_Sampler from this class. Fixed in 5cf5d274af